libs/api/domains/regulations-admin/src/lib/graphql/regulationsAdmin.resolver.ts

```diff
@@ -268,12 +265,12 @@ export class RegulationsAdminResolver {
       return null
     }
 
-    const regulation = await this.regulationsAdminApiService.getDraftRegulation(
+    const draftRegulation = await this.regulationsAdminApiService.getDraftRegulation(
       input.regulationId,
```

Comment:
Calling this `regulationId` is very confusing.   I believe it's actually a `draftId` ..no?